scheduler/bin/run-docker.sh

```diff
@@ -31,7 +31,7 @@ fi
 SCHEDULER_DIR="$( dirname ${DIR} )"
 SCHEDULER_EXECUTOR_DIR=${SCHEDULER_DIR}/resources/public
 EXECUTOR_NAME=cook-executor
-COOK_EXECUTOR_COMMAND="mkdir .cook-executor && ./${EXECUTOR_NAME}"
+COOK_EXECUTOR_COMMAND="./${EXECUTOR_NAME}/${EXECUTOR_NAME}"
```

Comment:
Maybe we should still make this a hidden file? (It seems like the consensus last week was that we don't want to clutter up the user's sandbox directory with files that they're not expecting.)
```
"./.${EXECUTOR_NAME}/${EXECUTOR_NAME}"
~~~^
```
I think that would require changing the build script to rename the folder. But if you guys think users won't be bothered by the presence of a `cook-executor` directory in their sandbox, then I'm happy leaving it like this.

Reply:
This path is used mainly for local development where the tar file is served by the scheduler and unzipped by the Mesos fetcher. I'd vote for avoiding the extra work for this scenario which is mainly used in development.

Reply:
Ah, right—this is in the run-docker script. That makes sense. No point in changing this for docker.